### PR TITLE
Fabo/3316 cross account creation

### LINKED ISSUE #3316: Create addresses for different networks
It should be possible to create an address for different networks apart from Cosmos.

### PR DESCRIPTION
**Pull request by faboweb**
Closes #3316, #3342, #3341

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
money on terra:
Bildschirmfoto von 2019-12-18 21-10-40

btw https://soju-faucet.terra.money/

**Comment by faboweb** [comment]
Test by adding `to.meta.feature !== "Session" &&` to line 21 in `router.js`

File: src/vuex/modules/keystore.js [hunk 1]
```diff
@@ -69,3 +69,37 @@ export default () => {
...
+    case "terra-mainnet": {
+      const addressCreator = await getCosmosAddressCreator(network)
+      return addressCreator(seedPhrase)
+    }
+    default:
+      throw new Error(
+        "Lunie doesn't support address creation for this network."
+      )
+  }
+}
```
Comment (Bitcoinera) [inline src/vuex/modules/keystore.js]: Neat!

**Comment by Bitcoinera** [comment]
> money on terra:
> 
> btw https://soju-faucet.terra.money/

I am having issues when sending... don't know if it is directly related with the new addresses. Looking into it right now.



Also happening now in `cosmos-testnet`:

**Comment by Bitcoinera** [comment]
It looks like we are missing something. The API returns "0" when querying for the balance:



And in Hubble...

**Comment by faboweb** [comment]
thanks for checking sending. I just checked account creation. will test this tomorrow.

**Comment by faboweb** [comment]
need to: prevent users from sending in with a wrong prefix

**Comment by Bitcoinera** [comment]
> need to: prevent users from sending in with a wrong prefix

Oops, good point. In `SendModal` it is only checking that the address sending to is valid bech32 address, nothing more

**Comment by faboweb** [comment]
hhaha I misstyped. But that is actually also a good check ^^

I actually meant: Prevent users to sign in with a wrong prefix. / or give them the option to instead log in to the chain that this address is used for.

**Comment by faboweb** [comment]
@Bitcoinera would you agree to merging this PR and creating a follow up which handles the issues with the balance in the API and the better checking for correct addresses?

**Comment by Bitcoinera** [comment]
> @Bitcoinera would you agree to merging this PR and creating a follow up which handles the issues with the balance in the API and the better checking for correct addresses?

For sure, yes. We can do that

**Comment by Bitcoinera** [comment]
I merged my first PR! Hehe, I feel so powerful now! :muscle:

**Pull request merged**